SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunner/Result.swift

```diff
@@ -6,23 +6,25 @@
 //
 
 func swift_func_takes_callback_with_result_arg(
-    arg: (RustResult<CallbackTestOpaqueRustType, String>) -> ()
+    arg: (RustResult<CallbackTestOpaqueRustType, String>) -> Void
 ) {
     arg(.Ok(CallbackTestOpaqueRustType(555)))
 }
 
 public class ResultTestOpaqueSwiftType {
     var num: UInt32
-    
+
     init(val: UInt32) {
         self.num = val
     }
-    
+
     func val() -> UInt32 {
         self.num
     }
 }
 
+extension AsyncRustFnReturnStruct: @unchecked Sendable {}
```

Comment:
Currently safe since this struct only contains a primitive https://github.com/chinedufn/swift-bridge/blob/636fa2774870c052a18f06d98ca5f6966c09bdd6/crates/swift-integration-tests/src/async_function.rs#L8-L11

But we'll still eventually want to replace all of these `@unchecked Sendable` with proper `swift-bridge` support for emitting `Sendable` impls as described in #269